docs/tutorials/cluster.md

```diff
@@ -322,12 +322,12 @@ You can copy your existing `coordinator-overlord` configs from the single-server
 
 #### Data
 
-Suppose we are migrating from a single-server deployment that had 32 CPU and 256GB RAM. In the old deployment, the following configurations for Historicals and MiddleManagers were applied:
+Suppose we are migrating from a single-server deployment that had 32 CPU and 256GiB RAM. In the old deployment, the following configurations for Historicals and MiddleManagers were applied:
 
 Historical (Single-server)
 
 ```
-druid.processing.buffer.sizeBytes=500000000
+druid.processing.buffer.sizeBytes=500MiB
```

Comment:
This change and the next one are lines from the single server small configuration file, so it might be clearer to leave them in their original form, so that they match the config files.

Reply:
This change makes the document consistent with the runtime.properties file. 

https://github.com/apache/druid/blob/af2ab98574582099557932e6a6370bf8dcdf3fab/examples/conf/druid/single-server/small/historical/runtime.properties#L27